guides/theming.md

```diff
@@ -110,28 +102,111 @@ as gulp-sass or grunt-sass). The simplest approach is to use the `node-sass` CLI
-secondary dark theme:
+You can create multiple themes for you application by including the `angular-material-theme` mixin
+multiple times, where each inclusion is gated by an additional CSS class.
+
+Remember to only ever include the `@mat-core` mixin only once; it should not be included for each 
+theme. 
+
+##### Example of defining multiple themes:
 ```scss
-.unicorn-dark-theme {
-  $dark-primary: mat-palette($mat-blue-grey);
-  $dark-accent:  mat-palette($mat-amber, A200, A100, A400);
-  $dark-warn:    mat-palette($mat-deep-orange);
+@import '~@angular/material/theming';
+// Plus imports for other components in your app.
+
+// Include the common styles for Angular Material. We include this here so that you only
+// have to load a single css file for Angular Material in your app.
+// **Be sure that you only ever include this mixin once!**
+@include mat-core();
+
+// Define the default theme (same as the example above).
+$candy-app-primary: mat-palette($mat-indigo);
+$candy-app-accent:  mat-palette($mat-pink, A200, A100, A400);
+$candy-app-theme:   mat-light-theme($candy-app-primary, $candy-app-accent);
+
+// Include the default theme styles.
+@include angular-material-theme($candy-app-theme);
+
 
-  $dark-theme: mat-dark-theme($dark-primary, $dark-accent, $dark-warn);
+// Define a alternate dark theme.
+$dark-primary: mat-palette($mat-blue-grey);
+$dark-accent:  mat-palette($mat-amber, A200, A100, A400);
+$dark-warn:    mat-palette($mat-deep-orange);
+$dark-theme:   mat-dark-theme($dark-primary, $dark-accent, $dark-warn);
 
+// Include the alternative theme styles inside of a block with a CSS class. You can make this
+// CSS class whatever you want. In this example, any component inside of an element with 
+// `.unicorn-dark-theme` will be affected by this alternate dark theme instead of the default theme. 
+.unicorn-dark-theme {
   @include angular-material-theme($dark-theme);
 }
 ```
 
-With this, any element inside of a parent with the `unicorn-dark-theme` class will use this
-dark theme.
+In the above example, any component inside of a parent with the `unicorn-dark-theme` class will use 
+the dark theme, while other components will fall back to the default `$candy-app-theme`.
+
+You can include as many themes as you like in this manner. You can also `@include` the 
+`angular-material-theme` in separate files and then lazily load them based on an end-user 
+interaction (how to lazily load the CSS assets will vary based on your application).
+
+It's important to remember, however, that the `mat-core` mixin should only ever be included _once_.
+
+##### Multiple themes and overlay-based components
+Since certain components (e.g. `dialog`) are inside of a global overlay container, the selector that
+determines the theme (such as the `.unicorn-dark-theme` example above), an additional step is needed
+to affect overlay-based components (menu, select, dialog, etc.).
```

Comment:
>Since certain components (e.g. `dialog`) are inside of a global overlay container, the selector that
  determines the theme (such as the `.unicorn-dark-theme` example above),

sounds like you're going to say something about that selector, instead we have this, which I'm not sure how it related to the above part:

> an additional step is needed to affect overlay-based components (menu, select, dialog, etc.).